prometheus_client/multiprocess_exporter.py

```diff
@@ -0,0 +1,44 @@
+import logging
+import sys
+import thread
+import time
+import traceback
+
```

Comment:
To use json in response body

```suggestion
import json

```